handling-null-values-in-avro-and-protobuf/kafka/README.md

```diff
@@ -0,0 +1,195 @@
+In this tutorial let's say we're tracking purchase events in Kafka with Confluent Cloud, each with an `item`, a `total_cost`, and a `customer_id`. 
+
+Let's say you're using an avro schema, and sometimes you want to set the `item` to null. How to adjust the schema to allow a null value? Let's get started. 
```

Comment:
suggest explaining why `item` might be null (... "sometimes you want to set `item` to null, e.g., if xyz") so that it feels a little more realistic. or, if null item is too much of a stretch, pick a different field where null would make sense. perhaps have a `store_id` field that gets set to null if a purchase is made online

Reply:
looks good with the donation case!